### PR TITLE
(Hotfix) AttributeError: 'coroutine' object has no attribute 'content' & Use poetry!

### PR DESCRIPTION
**Pull request by romanin-rf**

### CODE REVIEW

**Comment by romanin-rf** [comment]
I forgot to put `await` because of this, `corontine` is returned to the `response_auth` variable

**Comment by romanin-rf** [comment]
Use poetry to build, I've translated all of yours `setup.py` in `pyproject.toml`.

Read more about poetry here: https://python-poetry.org

**Comment by romanin-rf** [comment]
And to clarify a bit, your project is not supported in **python>=3.6**, **poetry** automatically checked the dependencies and gave out that your project is supported only with **python>=3.7**:
```
The current project's supported Python range (>=3.6) is not compatible with some of the required packages Python requirement:
  - aiofiles requires Python >=3.7, so it will not be satisfied for Python >=3.6,<3.7

Because no versions of aiofiles match >23.2.1
 and aiofiles (23.2.1) requires Python >=3.7, aiofiles is forbidden.
So, because vkpymusic depends on aiofiles (>=23.2.1), version solving failed.

  • Check your dependencies Python requirement: The Python requirement can be specified via the `python` or `markers` properties

    For aiofiles, a possible solution would be to set the `python` property to ">=3.7"

    https://python-poetry.org/docs/dependency-specification/#python-restricted-dependencies,
    https://python-poetry.org/docs/dependency-specification/#using-environment-markers
```

**Comment by issamansur** [comment]
Wow, thank you very much! I'll take a look. 

At first glance: 
1) Bug/typo with `await` in `TRAsync` - that's right, thank you, it's a gross mistake. 
2) Didn't work with `poetry`, so I'll look into it later. 
3) The last uploaded version on the `pip` does not contain `ServiceAsync` and `aiofiles`, which means it works fine on versions of `python 3.6-3.7` :) 
That's why I didn't push the latest commits to the linked `pip` library

**Comment by romanin-rf** [comment]
That's not the problem here. The same module simply cannot be installed on `python 3.6`, since it does not provide support for `python 3.6`. **PIP** will check the minimum supported version of python for child modules and return an error.

**Comment by romanin-rf** [comment]
`poetry` - is like an superstructure to **setuptools**, it is much more convenient and is used in many popular projects.

**Comment by romanin-rf** [comment]
> Wow, thank you very much! I'll take a look.
> 
> At first glance:
> 
> 1. Bug/typo with `await` in `TRAsync` - that's right, thank you, it's a gross mistake.
> 2. Didn't work with `poetry`, so I'll look into it later.
> 3. The last uploaded version on the `pip` does not contain `ServiceAsync` and `aiofiles`, which means it works fine on versions of `python 3.6-3.7` :)
>    That's why I didn't push the latest commits to the linked `pip` library

But you're going to update it in the future, aren't you?
`Python 3.6` has not been relevant for a long time, canceling its support will simply make it easier to maintain the code.

**Review by romanin-rf — COMMENTED** [review]
I just noticed :)

**Comment by issamansur** [comment]
> But you're going to update it in the future, aren't you? `Python 3.6` has not been relevant for a long time, canceling its support will simply make it easier to maintain the code.

Yes, you're definitely right

File: pyproject.toml
Comment (issamansur) [inline pyproject.toml:1]: Good work!

File: vkpymusic/TokenReceiverAsync.py
Comment (issamansur) [inline vkpymusic/TokenReceiverAsync.py:1]: Thanks for this)

**Review by issamansur — APPROVED** [review]
You did a good job, thank you!

P. S. I still haven't fully figured out `Poetry` due to my busy schedule, but according to the examples from the official documentation, your `pyproject.toml` looks great!

**Comment by romanin-rf** [comment]
Always happy to help!

**Pull request merged**